activation/activation_test.go

```diff
@@ -51,7 +51,7 @@ var (
 	postProver       = &postProverClientMock{}
 	npst             = NewNIPSTWithChallenge(&chlng, poetRef)
 	commitment       = &types.PostProof{
-		//Identity:     []byte(nil),
+		// Identity:     []byte(nil),
```

Comment:
remove commment?

Reply:
Sure, I just used the IDE's auto-format, didn't actually look at every line, will remove all of these now.